yt_dlp/extractor/nytimes.py

```diff
@@ -124,139 +88,348 @@ class NYTimesIE(NYTimesBaseIE):
-            'ext': 'mov',
-            'timestamp': 1479383008,
-            'uploader': 'By SHAW LASH, ADAM SAEWITZ and JAMES HERRON',
-            'title': 'Cranberry Tart',
-            'upload_date': '20161117',
-            'description': 'If you are a fan of lemon curd or the classic French tarte au citron, you will love this cranberry version.',
+            'id': '13-how-to-cook-a-turkey',
+            'title': 'How to Cook a Turkey',
+            'description': 'md5:726cfd3f9b161bdf5c279879e8050ca0',
         },
+        'playlist_count': 2,
     }, {
-        'url': 'https://cooking.nytimes.com/guides/13-how-to-cook-a-turkey',
-        'md5': '4b2e8c70530a89b8d905a2b572316eb8',
+        # single video example
+        'url': 'https://cooking.nytimes.com/guides/50-how-to-make-mac-and-cheese',
+        'md5': '64415805fe0b8640fce6b0b9def5989a',
         'info_dict': {
-            'id': '100000003951728',
-            'ext': 'mov',
-            'timestamp': 1445509539,
-            'description': 'Turkey guide',
-            'upload_date': '20151022',
-            'title': 'Turkey',
+            'id': '100000005835845',
+            'ext': 'mp4',
+            'title': 'How to Make Mac and Cheese',
+            'description': 'md5:b8f2f33ec1fb7523b21367147c9594f1',
+            'duration': 9510,
+            'creator': 'Alison Roman',
+            'thumbnail': r're:https?://\w+\.nyt.com/images/.*\.jpg',
         }
+    }, {
+        'url': 'https://cooking.nytimes.com/guides/20-how-to-frost-a-cake',
+        'md5': '64415805fe0b8640fce6b0b9def5989a',
+        'info_dict': {
+            'id': '20-how-to-frost-a-cake',
+            'title': 'How to Frost a Cake',
+            'description': 'md5:a31fe3b98a8ce7b98aae097730c269cd',
+        },
+        'playlist_count': 8,
     }]
 
+    _TOKEN = 'MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAuNIzKBOFB77aT/jN/FQ+/QVKWq5V1ka1AYmCR9hstz1pGNPH5ajOU9gAqta0T89iPnhjwla+3oec/Z3kGjxbpv6miQXufHFq3u2RC6HyU458cLat5kVPSOQCe3VVB5NRpOlRuwKHqn0txfxnwSSj8mqzstR997d3gKB//RO9zE16y3PoWlDQXkASngNJEWvL19iob/xwAkfEWCjyRILWFY0JYX3AvLMSbq7wsqOCE5srJpo7rRU32zsByhsp1D5W9OYqqwDmflsgCEQy2vqTsJjrJohuNg+urMXNNZ7Y3naMoqttsGDrWVxtPBafKMI8pM2ReNZBbGQsQXRzQNo7+QIDAQAB'
+    _DNS_NAMESPACE = uuid.UUID('36dd619a-56dc-595b-9e09-37f4152c7b5d')
+    _GRAPHQL_QUERY = '''query VideoQuery($id: String!) {
+  video(id: $id) {
+    ... on Video {
+      bylines {
+        renderedRepresentation
+      }
+      duration
+      promotionalHeadline
+      promotionalMedia {
+        ... on Image {
+          crops {
+            name
+            renditions {
+              name
+              width
+              height
+              url
+            }
+          }
+        }
+      }
+      renditions {
+        type
+        width
+        height
+        url
+        bitrate
+      }
+      summary
+    }
+  }
+}'''
+
+    def _build_playlist(self, media_items):
+        for media_id in media_items:
```

Comment:
imo it's more clear to just call them `media_ids`
```suggestion
    def _build_playlist(self, media_ids):
        for media_id in media_ids:
```